math/vc_aos/include/algebra/math/impl/vc_aos_transform3.hpp

```diff
@@ -224,27 +239,27 @@ struct transform3 {
     array_type<16> submatrix;
     for (unsigned int irow = 0; irow < 3; ++irow) {
       for (unsigned int icol = 0; icol < 3; ++icol) {
-        submatrix[icol + irow * 4] = element_getter()(_data, irow, icol);
+        submatrix[icol + irow * 4] = _data[icol][irow];
```

Comment:
Can't we use an appropriate element getter here still?

Reply:
I don't really see what that gains us here, when we can access the matrix natively the same way the getter does anyway? 

From what I can see, the getters are needed in generic code that can be shared by multiple plugins. In all other plugins except the cmath plugin we also access the data natively here instead of through an extra layer. And I find that easier to undestand tbh